controllers/isValidLar/type.js

```diff
@@ -5,7 +5,7 @@ var EngineService = require('../../services/EngineService');
 module.exports = function(router) {
 
     /**
-     * @api {post} /isValidLar/type/:activityYear/:editType isValidLar
+     * @api {post} /isValidLar/type/:activityYear/:editType type
```

Comment:
This is kind of out of scope for this pull, however, after looking at this, I didn't catch this when you implemented.. does it really make sense to have this as `/type/:year/:editType`?  Shouldn't the `editType` appear directly after `type`? And for that matter, why not just `/:editType/:year`? is `type` really necessary?